Creating a new team failed

When I going to create a new team in http://localhost:5173/dashboard/teams, below error occurred.

```
POST http://localhost:8000/api/v1/teams 405 (Method Not Allowed)
handleCreateTeam @ TeamsPage.tsx:183
await in handleCreateTeam
callCallback2 @ chunk-YQ5BCTVV.js?v=203a1d97:3680
invokeGuardedCallbackDev @ chunk-YQ5BCTVV.js?v=203a1d97:3705
invokeGuardedCallback @ chunk-YQ5BCTVV.js?v=203a1d97:3739
invokeGuardedCallbackAndCatchFirstError @ chunk-YQ5BCTVV.js?v=203a1d97:3742
executeDispatch @ chunk-YQ5BCTVV.js?v=203a1d97:7046
processDispatchQueueItemsInOrder @ chunk-YQ5BCTVV.js?v=203a1d97:7066
processDispatchQueue @ chunk-YQ5BCTVV.js?v=203a1d97:7075
dispatchEventsForPlugins @ chunk-YQ5BCTVV.js?v=203a1d97:7083
(anonymous) @ chunk-YQ5BCTVV.js?v=203a1d97:7206
batchedUpdates$1 @ chunk-YQ5BCTVV.js?v=203a1d97:18966
batchedUpdates @ chunk-YQ5BCTVV.js?v=203a1d97:3585
dispatchEventForPluginEventSystem @ chunk-YQ5BCTVV.js?v=203a1d97:7205
dispatchEventWithEnableCapturePhaseSelectiveHydrationWithoutDiscreteEventReplay @ chunk-YQ5BCTVV.js?v=203a1d97:5484
dispatchEvent @ chunk-YQ5BCTVV.js?v=203a1d97:5478
dispatchDiscreteEvent @ chunk-YQ5BCTVV.js?v=203a1d97:5455
TeamsPage.tsx:217 Error creating team: Error: Method Not Allowed
    at handleCreateTeam (TeamsPage.tsx:195:15)
handleCreateTeam @ TeamsPage.tsx:217
await in handleCreateTeam
callCallback2 @ chunk-YQ5BCTVV.js?v=203a1d97:3680
invokeGuardedCallbackDev @ chunk-YQ5BCTVV.js?v=203a1d97:3705
invokeGuardedCallback @ chunk-YQ5BCTVV.js?v=203a1d97:3739
invokeGuardedCallbackAndCatchFirstError @ chunk-YQ5BCTVV.js?v=203a1d97:3742
executeDispatch @ chunk-YQ5BCTVV.js?v=203a1d97:7046
processDispatchQueueItemsInOrder @ chunk-YQ5BCTVV.js?v=203a1d97:7066
processDispatchQueue @ chunk-YQ5BCTVV.js?v=203a1d97:7075
dispatchEventsForPlugins @ chunk-YQ5BCTVV.js?v=203a1d97:7083
(anonymous) @ chunk-YQ5BCTVV.js?v=203a1d97:7206
batchedUpdates$1 @ chunk-YQ5BCTVV.js?v=203a1d97:18966
batchedUpdates @ chunk-YQ5BCTVV.js?v=203a1d97:3585
dispatchEventForPluginEventSystem @ chunk-YQ5BCTVV.js?v=203a1d97:7205
dispatchEventWithEnableCapturePhaseSelectiveHydrationWithoutDiscreteEventReplay @ chunk-YQ5BCTVV.js?v=203a1d97:5484
dispatchEvent @ chunk-YQ5BCTVV.js?v=203a1d97:5478
dispatchDiscreteEvent @ chunk-YQ5BCTVV.js?v=203a1d97:5455
AuthContext.tsx:213 Auth state changed: SIGNED_IN
```

--- a/backend/app/api/v1/team/teams.py
+++ b/backend/app/api/v1/team/teams.py
@@ -46,10 +46,20 @@ def convert_team_to_dict(team, include_members: bool = False) -> Dict:
         "team_size": team.team_size or 0,
     }
 
-    # Only include members if requested and available
-    if include_members and hasattr(team, "members") and team.members:
+    # Check if members are already loaded to avoid lazy loading
+    try:
+        members_loaded = (hasattr(team, "_sa_instance_state") and 
+                          hasattr(team._sa_instance_state, "attrs") and 
+                          hasattr(team._sa_instance_state.attrs, "members") and 
+                          (getattr(team._sa_instance_state.attrs.members, "loaded", False) or 
+                           team._sa_instance_state.attrs.members.loaded_value is not None))
+    except Exception as e:
+        logger.error(f"Error checking if members are loaded: {str(e)}")
+        members_loaded = False
+    
+    if include_members and members_loaded and hasattr(team, "members"):
         try:
-            # Explicitly load and convert members to avoid lazy loading issues
+            # Convert members to dicts to avoid further lazy loading
             member_list = []
             for member in team.members:
                 member_dict = {
@@ -70,7 +80,7 @@ def convert_team_to_dict(team, include_members: bool = False) -> Dict:
             logger.error(f"Error loading team members: {str(e)}")
             team_dict["members"] = []
     else:
-        team_dict["members"] = None
+        team_dict["members"] = []
 
     return team_dict
 

--- a/backend/app/services/team/teams.py
+++ b/backend/app/services/team/teams.py
@@ -91,7 +91,11 @@ class TeamService:
 
                 db.add(team_member)
                 await db.commit()
-                await db.refresh(team)
+                
+                # Explicitly load the team with its members to avoid lazy loading issues
+                query = select(Team).where(Team.id == team.id).options(selectinload(Team.members))
+                result = await db.execute(query)
+                team = result.scalars().first()
 
                 logger.info(
                     f"Auto-created team '{team.name}' (ID: {team.id}) for user {user_id}"
@@ -230,12 +234,16 @@ class TeamService:
 
             db.add(team_member)
             await db.commit()
-            await db.refresh(team)
+            
+            # Explicitly load the team with its members to avoid lazy loading issues
+            query = select(Team).where(Team.id == team.id).options(selectinload(Team.members))
+            result = await db.execute(query)
+            team_with_members = result.scalars().first()
 
             logger.info(
                 f"Created team '{team.name}' (ID: {team.id}) for user {user_id}"
             )
-            return team
+            return team_with_members
 
         except IntegrityError as e:
             logger.error(f"Integrity error creating team: {str(e)}")

--- a/frontend/src/pages/team/TeamsPage.tsx
+++ b/frontend/src/pages/team/TeamsPage.tsx
@@ -180,7 +180,7 @@ const TeamsPage: React.FC = () => {
       } = await supabase.auth.getSession()
       const token = session?.access_token
 
-      const response = await fetch(`${env.apiUrl}/v1/teams`, {
+      const response = await fetch(`${env.apiUrl}/teams/`, {
         method: 'POST',
         credentials: 'include',
         headers: {
@@ -305,7 +305,7 @@ const TeamsPage: React.FC = () => {
       } = await supabase.auth.getSession()
       const token = session?.access_token
 
-      const response = await fetch(`${env.apiUrl}/v1/teams/${teamId}`, {
+      const response = await fetch(`${env.apiUrl}/teams/${teamId}/`, {
         method: 'DELETE',
         credentials: 'include',
         headers: {

--- a/frontend/src/pages/team/TeamsPage.tsx
+++ b/frontend/src/pages/team/TeamsPage.tsx
@@ -180,7 +180,7 @@ const TeamsPage: React.FC = () => {
       } = await supabase.auth.getSession()
       const token = session?.access_token
 
-      const response = await fetch(`${env.apiUrl}/teams`, {
+      const response = await fetch(`${env.apiUrl}/v1/teams`, {
         method: 'POST',
         credentials: 'include',
         headers: {
@@ -305,7 +305,7 @@ const TeamsPage: React.FC = () => {
       } = await supabase.auth.getSession()
       const token = session?.access_token
 
-      const response = await fetch(`${env.apiUrl}/teams/${teamId}`, {
+      const response = await fetch(`${env.apiUrl}/v1/teams/${teamId}`, {
         method: 'DELETE',
         credentials: 'include',
         headers: {

